iwyu_output.cc

```diff
@@ -537,6 +538,19 @@ void IwyuFileInfo::AddForwardDeclare(const clang::NamedDecl* fwd_decl,
            << internal::GetQualifiedNameAsString(fwd_decl) << "\n";
 }
 
+void IwyuFileInfo::AddUsingDecl(const clang::UsingDecl* using_decl) {
```

Comment:
You shouldn't need to namespace-qualify `UsingDecl` here, there was a `using` of it above.